examples/worlds/shapes_population.sdf.erb

```diff
@@ -0,0 +1,275 @@
+<?xml version="1.0" ?>
+
```

Comment:
same here, can you add a description with the command for generating the sdf file?